### PR TITLE
Module init

### PR DESCRIPTION
**Pull request by mwarkentin**
### Notes

* Terraform docs will be auto-generated by a Github action

### Testing

* Run `make alpha scope=<major|minor|patch>` to tag and publish an alpha version
* Update the `version = "..."` of your module usage to your published alpha version (eg. `version = "v2.2.0-alpha.1+9f2630a"`)
* Run `make plan` in the keller terraform workspace

### Checklist

* [x] Run `make changelog scope=<major|minor|patch>`
* [x] CHANGELOG.md: Adjust change notes as necessary

### Publishing

* [ ] Merge this Pull Request
* [ ] Run `git checkout master; git pull; make publish scope=<major|minor|patch>`

### CODE REVIEW

**Comment by mwarkentin** [comment]
Successfully created a chatbot slack configuration (locally):

File: README.md [hunk 1]
```diff
@@ -15,14 +54,27 @@
...
+| iam\_role\_arn | The ARN of the IAM role that defines the permissions for AWS Chatbot. This is a user-defined role that AWS Chatbot will assume. This is not the service-linked role. For more information, see IAM Policies for AWS Chatbot. | `any` | n/a | yes |
+| logging\_level | Specifies the logging level for this configuration. This property affects the log entries pushed to Amazon CloudWatch Logs. Logging levels include ERROR, INFO, or NONE. | `string` | `"ERROR"` | no |
+| slack\_channel\_id | The ID of the Slack channel. To get the ID, open Slack, right click on the channel name in the left pane, then choose Copy Link. The channel ID is the 9-character string at the end of the URL. For example, ABCBBLZZZ. | `any` | n/a | yes |
+| slack\_workspace\_id | The ID of the Slack workspace authorized with AWS Chatbot. To get the workspace ID, you must perform the initial authorization flow with Slack in the AWS Chatbot console. Then you can copy and paste the workspace ID from the console. For more details, see steps 1-4 in Setting Up AWS Chatbot with Slack in the AWS Chatbot User Guide. | `any` | n/a | yes |
```
Comment (dawilliams-wave) [inline README.md:70]: Is that referring to this guide? https://docs.aws.amazon.com/chatbot/latest/adminguide/getting-started.html

Comment (mwarkentin) [reply README.md:70]: Yep, I just copied the descriptions from the cloudformation page: https://docs.aws.amazon.com/chatbot/latest/adminguide/setting-up.html#Setup_intro

Comment (mwarkentin) [reply README.md:70]: Wonder what happens if we stick markdown in there..

Comment (mwarkentin) [reply README.md]: @dawilliams-wave looks like it works!

**Pull request merged**